.eleventyignore

```diff
@@ -1 +1,2 @@
 README.md
+functions/
```

Comment:
Cloudflare needs this & runs it as serverless functions, but it's not part of the Eleventy build.